src/components/ContextualMenu/ContextualMenu.tsx

```diff
@@ -148,18 +148,18 @@ export class ContextualMenu extends React.Component<IContextualMenuProps, IConte
 
   public render() {
     let { className,
-          items,
-          isBeakVisible,
-          labelElementId,
-          targetElement,
-          id,
-          targetPoint,
```

Comment:
it might be worth updating the example to render an anchor and button next to each other so that it can be demonstrated.